### DISCUSSION TITLE
Update timeline?

### DISCUSSION THREAD

**Discussion by tonit** [gitops-working-group/gitops-working-group#68] [Q&A] [answered]
On the readme it says:

>The goal of the GitOps Working Group’s founders is to: firstly, form the group during Q4 of 2020 and secondly, to establish V1 of the GitOps Manifesto by the end of March, 2021. If you are interested in keeping abreast of any new developments, please visit follow this repository.

shall we update that? What would be a good timeline?
I'd include the day-zero event at kubecon, which maybe could be a good goal for having "princples version 1.0" ?
So an update could say:

>The goal of the GitOps Working Group is to: Establish the CNCF Sandbox Project and have a version 1.0 manifesto released together with a successful Day-Zero Event at Kubecon 2021 in May 2021.  If you are interested in keeping abreast of any new developments, please visit follow this repository.

**Comment by scottrigby** [comment, marked as answer]
I updated the README [Timeline section](https://github.com/gitops-working-group/gitops-working-group/blob/main/README.md#timeline) as part of https://github.com/gitops-working-group/gitops-working-group/pull/84.
